esp/templates/email/default_email_html.txt

```diff
@@ -0,0 +1,5 @@
+<html>
+{% autoescape off %}
+{{ msgbdy }}
+{% endautoescape %}
```

Comment:
If we have a nice template, making a 1-click unsubscribe wouldn't be too hard -- there might even be a library that will do most of the work for us.  But in any case, we can do that after this is merged.